cmd/kube-apiserver/app/aggregator.go

```diff
@@ -95,7 +95,10 @@ func createAggregatorConfig(
 	etcdOptions.StorageConfig.Paging = utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIListChunking)
 	etcdOptions.StorageConfig.Codec = aggregatorscheme.Codecs.LegacyCodec(v1.SchemeGroupVersion, v1beta1.SchemeGroupVersion)
 	etcdOptions.StorageConfig.EncodeVersioner = runtime.NewMultiGroupVersioner(v1.SchemeGroupVersion, schema.GroupKind{Group: v1beta1.GroupName})
-	genericConfig.RESTOptionsGetter = &genericoptions.SimpleRestOptionsFactory{Options: etcdOptions}
+	genericConfig.RESTOptionsGetter = &genericoptions.StorageFactoryRestOptionsFactory{
```

Comment:
I'm wondering if these should call `etcdOptions.ApplyTo(genericConfig)` and we should fix the transformer multi-loading in ApplyTo/ApplyWithStorageFactoryTo